.gitignore

```diff
@@ -1,9 +1,2 @@
-CMakeCache.txt
-CMakeFiles
-CMakeScripts
-Testing
-Makefile
-cmake_install.cmake
-install_manifest.txt
-compile_commands.json
-CTestTestfile.cmake
+.idea
+build
```

Comment:
I believe it's not good idea ignore `build`. Better exclude files by extension: 
 - *.dll
 - *.exe
 - *.o
 - *.so

Remove `build` or add exceptions which we really must exclude.